src/app/tab3/tab3.page.html

```diff
@@ -13,6 +13,12 @@
       <ion-title size="large">Journal Entries</ion-title>
     </ion-toolbar>
   </ion-header>
+        <!-- fab placed to the top end -->
+        <ion-fab vertical="top" horizontal="end" slot="fixed">
+          <ion-fab-button>
```

Comment:
ensure the button is wired up to the open modal function. You can probably just call openModal() with no arguments.